spring-xd-dirt/src/test/java/org/springframework/xd/dirt/integration/kafka/KafkaTestMessageBus.java

```diff
@@ -28,31 +36,43 @@
  * Creates a bus that uses a test {@link TestKafkaCluster kafka cluster}.
  *
  * @author Eric Bottard
+ * @author Marius Bogoevici
  */
 public class KafkaTestMessageBus extends AbstractTestMessageBus<KafkaMessageBus> {
 
+	public KafkaTestMessageBus(KafkaTestSupport kafkaTestSupport) {
+		this(kafkaTestSupport, getCodec());
+	}
 
-	private TestKafkaCluster kafkaCluster;
 
-	public KafkaTestMessageBus(MultiTypeCodec<Object> codec) {
-		kafkaCluster = new TestKafkaCluster();
+	public KafkaTestMessageBus(KafkaTestSupport kafkaTestSupport, MultiTypeCodec<Object> codec) {
 
-		KafkaMessageBus messageBus = new KafkaMessageBus(kafkaCluster.getKafkaBrokerString(),
-				kafkaCluster.getZkConnectString(), codec);
-		GenericApplicationContext context = new GenericApplicationContext();
-		context.refresh();
-		messageBus.setApplicationContext(context);
-		this.setMessageBus(messageBus);
+		try {
+			ZookeeperConnect zookeeperConnect = new ZookeeperConnect();
+			zookeeperConnect.setZkConnect(kafkaTestSupport.getZkconnectstring());
```

Comment:
Shouldn't the method `getZkconnectstring` name be well-formated in camle style ?